src/wallet/crypter.cpp

```diff
@@ -261,6 +287,67 @@ bool CCryptoKeyStore::Unlock(const CKeyingMaterial& vMasterKeyIn)
     return true;
 }
 
+bool CCryptoKeyStore::SetHDSeed(const HDSeed& seed)
+{
+    {
```

Comment:
Question: I see this double `{ {` from time to time before calling lock. Why do we do this. In this case does this cause us to unlock before having returned false rather than after?

Reply:
The style is inherited from upstream.  The { scopes the lock to a block of code. So if there were code before or after which didn't require the lock, it could be added easily and it would also be easy to see which section of code in the function required a lock.